sched/pthread/pthread_mutexinit.c

```diff
@@ -124,11 +117,20 @@ int pthread_mutex_init(FAR pthread_mutex_t *mutex,
 #endif
 
 #ifndef CONFIG_PTHREAD_MUTEX_UNSAFE
-      /* Initial internal fields of the mutex */
 
       mutex->flink  = NULL;
+
+#ifdef CONFIG_PTHREAD_MUTEX_BOTH
+      /* Initial internal fields of the mutex */
+
       mutex->flags  = (robust == PTHREAD_MUTEX_ROBUST ?
```

Comment:
If you removed `uint8_t robust` at lines 72 and 74, then what is checked here?

Reply:
Yes, I add uint8_t robust when CONFIG_PTHREAD_MUTEX_BOTH is enable.

Reply:
@pkarashchenko  please review again

Reply:
what is the exact error reported here by coverity?